apis/pipelines/v1alpha6/experiment_conversion.go

```diff
@@ -0,0 +1,3 @@
+package v1alpha6
+
+func (*Experiment) Hub() {}
```

Comment:
assuming this is required for the conversion even if its not doing anything?

Reply:
Yes, thats the hub